lib/active_resource.rb

```diff
@@ -30,6 +30,9 @@
 require "active_resource/exceptions"
 require "active_resource/version"
 
+# Only necessary as long as Ruby 2.6 compatibility is kept
+require "ruby2_keywords"
```

Comment:
Should this be wrapped in a condition for Ruby < 2.7? It seems to be a no-op in Ruby 3, but I don't know if there is a performance impact to be concerned about:

```
irb(main):001> require "ruby2_keywords"
=> false
```